app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -211,12 +211,13 @@ def post_refresh(ems, ems_refresh_start_time)
           _log.info("#{log_ems_target} Performing post-refresh operations for #{klass} instances...Complete")
         end
       end
-      private
```

Comment:
I don't think we want to remove this `private`.

Is this intentionally making these public?

- `group_targets_by_ems`
- `refresher_type`
- `format_ems_for_logging`

Reply:
No, it wasn't intentional. My mistake.